jdaviz/core/template_mixin.py

```diff
@@ -2912,20 +2922,28 @@ def _on_viewer_layer_changed(self, msg=None):
         self.sync = {**self.sync,
                      'in_subscribed_states': in_subscribed_states,
                      'icons': icons,
-                     'mixed': len(np.unique(current_glue_values, axis=0)) > 1}
-
+                     'mixed': self.is_mixed(current_glue_values)}
         if len(current_glue_values):
             # sync the initial value of the widget, avoiding recursion
             self._on_glue_value_changed(current_glue_values[0])
 
+    def is_mixed(self, glue_values):
+        if len(glue_values) and isinstance(glue_values[0], dict):
+            # If we want to expose dictionary inputs in the plot options UI, 
+            # this will need to be updated to check if any of the dictionaries 
+            # in the list are not exact matches
+            mixed = None
+            return
```

Comment:
```suggestion
            return None
```

Reply:
Aren't `return` and `return None` the same thing?

Reply:
yes, this will work as is, but the function is supposed to return the value for mixed, so I think it makes sense to explicitly state it (whereas the existing sets a variable that is never used and so could be easy to break if we change `mixed` to something else but don't update the return statement.